sopel/irc/abstract_backends.py

```diff
@@ -26,6 +26,17 @@ def is_connected(self):
         """
         raise NotImplementedError
 
+    def on_irc_error(self, pretrigger):
+        """Action to perform when the server send an error event.
+
+        :param pretrigger: Pretrigger object with the error event.
```

Comment:
Ditto the other `:param:` note.

```suggestion
        :param pretrigger: PreTrigger object with the error event
```